mis_builder/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     'name': 'MIS Builder',
-    'version': '9.0.2.0.2',
+    'version': '10.0.2.0.2',
```

Comment:
@pedrobaeza since the 10.0 and 9.0 are identical (except very minor changes in the source code) we plan to maintain feature parity between the two versions for some time. And we would like to use the version number to express the feature level of each series. So in this case I would like to have 10.0.2.0.3.

Do you have any objection for that scheme?